### ISSUE TITLE
How to merge Profile in Java home?

### ISSUE THREAD

**Issue by Vinitdp** [Vinitdp/Profile#1]
```
    <%@ page language="java" contentType="text/html; charset=ISO-8859-1"
    pageEncoding="ISO-8859-1"%>
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"     "http://www.w3.org/TR/html4/loose.dtd">
```

<html>
<head>
<link rel="stylesheet" type="text/css" href="css/combined.css">
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<title>Insert title here</title>
</head>

<script src="js/jquery.min.js"></script>

<body class="body" id="main_body_tag">

<div class="main_container first_page" style="/* margin-left: 134px;  */display: block; padding-bottom: 20px;">
 <div class="top_nav" style="margin-left:15px">
 
          <div id="headerMenuNavigation" class="top_nav_liner">
            <ul class="tabs_ul" style="width: 1245px;">
                
            
                <li id="main_tab_5" class="last_node selected">
                    <a href="home.jsp"></a>
                    
                    <p>Merge Records</p>
                </li>
                
            
                
                
            </ul>
            <div class="clr_l"></div>
          </div>
        </div>
                <div id="main_body_inbox" class="main_body" style="display: block;">

</div>
 

<div id="details_screen">
</div>

```
    <div id="announcement_container" class="leftMenuSelected" style="margin-right:auto;margin-left: 40%">
    <div class="dataTables_filter" id="announcementTable_filter"><form name="searchFilter" id="searchFilterId" method="get" action=getPersonAjax.jsp><input type="search" class="searchTextClass" id="personSearch1" name="personSearchF"  placeholder="First Name" aria-controls="announcementTable"><input type="search" class="searchTextClass" id="personSearch2" name="personSearchL"  placeholder="Last name" aria-controls="announcementTable">
    <input type="submit"   value="search"/></form></div>
```

</div></div>
</body>
</html>
